customer_service_bot/README.md

```diff
@@ -5,12 +5,12 @@
 Customer service bot built on `DFF`. Uses telegram as an interface.
 This bot is designed to answer any type of user questions in a limited business domain (book shop).
 
-* [DNNC](https://github.com/salesforce/DNNC-few-shot-intent) model ([Zhang et al. 2020](https://arxiv.org/abs/2010.13009)) force is used for intent retrieval.
+* [DeepPavlov Intent Catcher](#) force is used for intent retrieval.
```

Comment:
Add a link to the intent catcher.
I'm assuming the link is https://huggingface.co/ruthenian8/deeppavlov-intent-catcher-transformers.
If so, the readme file in that model card should include some info about it.